lib/commands/setViewportSize.js

```diff
@@ -8,12 +8,12 @@
  :setViewportSize.js
  client
  .setViewportSize({
-            width: 500,
-            height: 500
-        })
+    width: 500,
+    height: 500
+ })
  .windowHandleSize(function(err, res) {
-            console.log(res.value); // outputs: "{ width: 500, height: 602 }"
-        })
+    console.log(res.value); // outputs: "{ width: 500, height: 602 }"
```

Comment:
should be `console.log(res.value); // outputs: "{ width: 500, height: 500 }"` then